src/panda_sdl/frontend_sdl.cpp

```diff
@@ -35,6 +35,7 @@ FrontendSDL::FrontendSDL() : keyboardMappings(InputMappings::defaultKeyboardMapp
 		SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_CORE);
 		SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, config.rendererType == RendererType::Software ? 3 : 4);
 		SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, config.rendererType == RendererType::Software ? 3 : 1);
+		SDL_GL_SetAttribute(SDL_GL_SHARE_WITH_CURRENT_CONTEXT, 1);
```

Comment:
Should not be enabled when not actually using async shaders

Reply:
Are you sure? This needs to be set before creating the main opengl context, and it might be the case you want to make the shaderMode configurable at runtime in the future.

Reply:
Very sure, some vendors probably have terribly broken shared context implementations and there's no reason to go through all that if not actually using it. Being able to configure shader mode at runtime between ubershader and specialized shaders is fine (and supported in my shader decompiler branch) but hybrid shaders are a whole other mess.

Reply:
Done